packages/dds/sequence-sequential-id/src/SharedStringWithSequentialIdFactory.ts

```diff
@@ -0,0 +1,59 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import * as MergeTree from "@fluidframework/merge-tree";
+import { IChannelAttributes, IChannelServices,
+    IFluidDataStoreRuntime, IChannelFactory } from "@fluidframework/datastore-definitions";
+import { SharedString } from "@fluidframework/sequence";
+import { pkgVersion } from "./packageVersion";
+import { sharedStringWithSequentialIdMixin } from "./sharedStringWithSequentialId";
+
+export class SharedStringWithSequentialIdFactory implements IChannelFactory {
+    // TODO rename back to https://graph.microsoft.com/types/mergeTree/string once paparazzi is able to dynamically
```

Comment:
Not sure what this comment refers to, but I bet it's not adding value, so I'd remove it.

Reply:
Removing comment